### PR TITLE
implement fix for ticket PLT-2060

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
This PR implement System Console > Server Logs, scroll to bottom of logs on page load and when reload the logs

#### Ticket Link
Github - https://github.com/mattermost/platform/issues/4490
Jira - https://mattermost.atlassian.net/browse/PLT-2060

### CODE REVIEW

**Comment by jasonblais** [comment]
Thanks @cpanato! 

The changes work as specified in the GitHub issue and Jira ticket. Server logs table is now scrolled to the bottom upon reload.

-----

However, given the server logs table doesn't resize based on the window size, the page itself isn't scrolled to the bottom. Hence, you cannot actually see the latest server log messages:

image

Given it's beyond the scope of the original issue, I've created an improvement ticket here to resize the server logs table based on the window size, removing the need to have a scrollbar on the page itself: https://mattermost.atlassian.net/browse/PLT-4740

@cpanato If you're interested, the ticket is up for grabs :)

File: webapp/components/admin_console/logs.jsx [hunk 1]
```diff
@@ -26,6 +26,12 @@ export default class Logs extends React.Component {
         AsyncClient.getLogs();
     }
 
+    componentDidUpdate() {
+        // Scroll Down to get the latest logs
+        var node = ReactDOM.findDOMNode(this.refs.logPanel);
```
Comment (jwilander) [inline webapp/components/admin_console/logs.jsx]: Minor, but you should be able to just use `var node = this.refs.logPanel`. ReactDOM is scheduled to be deprecated so if you could switch it over that would be great

Comment (cpanato) [inline webapp/components/admin_console/logs.jsx]: ok, cool, will do that!

**Comment by cpanato** [comment]
yay :)

**Pull request merged**